docs/proto/proto-docs.md

```diff
@@ -17,6 +17,8 @@
     - [Query](#babylonlabs.babylon.v1beta1.Query)
   
 - [babylonlabs/babylon/v1beta1/tx.proto](#babylonlabs/babylon/v1beta1/tx.proto)
+    - [MsgInstantiateBabylonContracts](#babylonlabs.babylon.v1beta1.MsgInstantiateBabylonContracts)
```

Comment:
I think providing a message for instantiation is not without merit, but why not just instantiate the contracts during bootstrapping?

Since the contracts are fundamental for the SDK operation, I don't see the need to gate their instantiation through a specific protobuf endpoint.

Reply:
> why not just instantiate the contracts during bootstrapping?

I was thinking about this as well, but decided to not do this (at least for this PR):

- This makes it mandatory to have these wasm binaries in hand when executiong `babylond` to start the chain. Otherwise we would need to somehow wire the wasm binaries into the `babylond `binary, which is not ideal.
- It's possible that one wants to invoke `MsgInstantiateBabylonContracts` after the chain has been running for a while, e.g., 
  - the chain has just passed a gov prop for integrating with Babylon; 
  - the chain does not want the old babylon contracts anymore, and wants to deploy a new set of babylon contracts, e.g., due to corrupted DB state
- Even if we want to instantiate all contracts upon starting the chain, this could be done in a separate PR. This separate PR basically executes the same logic of `MsgInstantiateBabylonContracts` handler upon starting the chain. Other things are not affected.

Wdyt?